### PR TITLE
Release version 1.4.22

### PR DESCRIPTION
**Pull request by hakontro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -999,12 +999,15 @@ One of two additional options must be specified:
...
-  .option("id", sequenceId) // or you can use "externalId" option
+  .option("id", sequenceId) // or you can use the "externalId" option
   .load()
 
 // Insert the rows into another sequence using .save()
 import org.apache.spark.sql.functions._
+import org.apache.spark.sql.functions.lit
+
 df
+  .withColun("externalId", lit("my-sequence")) // Required when writing to support writing to multiple sequences
```
Comment (emelvarol-cognite) [inline README.md:1010]:
```suggestion
  .withColumn("externalId", lit("my-sequence")) // Required when writing to support writing to multiple sequences
```

Comment (hakontro) [reply README.md:1010]: True, that is better

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+# 1.4.23
+
+## Enhancements
+* SequenceRows are now written to one or multiple sequences by refering each rows `externalId`.
+
+# 1.4.22
+
+## Enhancements
+* Now uses the 1.4.6 version of the Scala SDK.
+
```
Comment (emelvarol-cognite) [inline CHANGELOG.md:10]: My bad I forgot to release this one 🤦 Should we skip this and not bump the version here?

Comment (hakontro) [reply CHANGELOG.md:10]: Good point, we can do that

**Review by emelvarol-cognite — APPROVED** [review]

**Pull request merged**